docs/content/config/best-practices/dkim_dmarc_spf.md

```diff
@@ -156,6 +156,10 @@ DKIM is currently supported by either OpenDKIM or Rspamd:
         use_esld = true;
         check_pubkey = true; # you want to use this in the beginning
 
+        selector = "mail";
+        path = "/tmp/docker-mailserver/opendkim/keys/$domain/$selector.private"; # this will automatically match keys for domains
+
+        # domain specific configurations can be provided below:
```

Comment:
Alternatively there is a simple [`selector_map` setting](https://rspamd.com/doc/modules/dkim_signing.html#using-maps) that references a file of `$domain $selector` pairs per line.

The `$domain` is used to lookup the `$selector` override (if any), similar to how the config here works. Assuming `selector_maps` is compatible with multiple selectors as well, then the only difference from this config below is configuring different `path` which we don't really need, and a similar setting `path_maps` supports.

Reply:
This feedback has been migrated to a separate TODO issue: https://github.com/docker-mailserver/docker-mailserver/issues/3778